packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -76,7 +76,7 @@ type BaseButtonCommonProps = {
   type?: 'button' | 'reset' | 'submit';
   isLoading?: boolean;
   accessibilityProps?: Partial<AccessibilityProps>;
-  variant?: 'primary' | 'secondary' | 'tertiary';
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'transparent';
```

Comment:
wouldn't having it as `color` instead of `variant` make things a bit easier? I can see most colors match the colors of variant tertiary so only thing you would have to change is default background color and icon color 🤔